documentation/modules/auxiliary/scanner/http/syncovery_linux_login.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+[Syncovery For Linux with Web-GUI](https://www.syncovery.com/download/linux/)
+
+This module attempts to brute-force valid login credentials for the Syncovery File Sync & Backup Software Web-GUI for Linux.
+The default credentials are checked by default.
+
+### Authors
+
+- Jan Rude (mgm security partners GmbH)
+
+### Platforms
+
+- Unix
+
+## Verification Steps
+
+1. Install the application
+2. Start msfconsole
+3. Do: `use modules/auxiliary/scanner/http/syncovery_linux_login`
+4. Do: `set RHOSTS <TARGET HOSTS>`
+5. Do: `run`
+6. On success you should get valid credentials.
+
```

Comment:
The `Options` section seems to be missing after `Verification Steps`. This section should list the specific options you defined for this modules with a description:
```
## Options

### USERNAME
<description>

### PASSWORD
<description>

### TARGETURI
<description>
```